packages/application-shell-connectors/package.json

```diff
@@ -40,6 +40,11 @@
   "dependencies": {
     "@babel/runtime-corejs3": "7.5.5",
     "@commercetools-frontend/constants": "14.1.1",
+    "apollo-cache-inmemory": "1.6.2",
+    "apollo-client": "2.6.3",
+    "apollo-link-http": "1.5.15",
+    "graphql": "14.4.0",
+    "graphql-tag": "^2.10.1",
```

Comment:
...without the `^`?

Reply:
It's the same as in app-shell actually...it's a bit weird though but those are actually the required dependencies of `react-apollo`, that's why I had to define them. The error I got if I didn't define them was something like this:

```
 Cannot find module 'apollo-cache-inmemory' from 'test-utils.js'

    However, Jest was able to find:
        './test-utils.js'
        './test-utils.spec.js'

    You might want to include a file extension in your import, or update your 'moduleFileExtensions', which is currently ['ts', 'tsx', 'js', 'jsx', 'json'].

    See https://jestjs.io/docs/en/configuration#modulefileextensions-array-string

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:230:17)
      at Object.<anonymous> (node_modules/src/test-utils.tsx:4:1)
```